games/ninswitch/game_simple_reset.py

```diff
@@ -1,11 +1,32 @@
 from surrortg import Game
```

Comment:
I'd keep the `game_simple.py` name. Creators can then optionally connect the reset GPIO pin.